server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -102,13 +102,13 @@ cics_into_set:                  INTO cics_data_area | SET cics_ref;
 cics_length_flength:            (LENGTH | FLENGTH) cics_data_area;
```

Comment:
At this moment, I am wondering if the "Helpers" (`cics_into_set`, `cics_length_flength`, `cics_maxlength`) are actually helping. I think inlining the rules and adjusting the checker will lead to more readable code.